src/sql/run.py

```diff
@@ -469,7 +488,15 @@ def run(conn, sql, config):
         else:
             txt = sqlalchemy.sql.text(statement)
             manual_commit = set_autocommit(conn, config)
-            result = conn.session.execute(txt)
+
+            is_custom_connection = Connection.is_custom_connection(conn)
+            if IS_SQLALCHEMY_ONE or is_custom_connection:
+                txt_ = str(txt)
```

Comment:
sqlalchemy.sql.text returns `TextClause` which can be used as an argument to an SQLAlchemy query but it won't work when we'll use a custom connection. 

Removed `IS_SQLALCHEMY_ONE` since it's now working for both (1 and 2).